Doc/library/tkinter.rst

```diff
@@ -67,13 +67,33 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1, sync=0, use=None)
+
+   Return a toplevel Tk widget, which is usually the main window of an
```

Comment:
This is a class, not a function. The word "return" doesn't look good to me here. What if just omit it?
```
Toplevel Tk widget, which is usually ...
```
See also the docstring.

Reply:
OK.

Reply:
I suggest "Create a top-level Tk widget ...".